addons/portal/controllers/portal.py

```diff
@@ -157,6 +157,17 @@ def home(self, **kw):
         values = self._prepare_portal_layout_values()
         return request.render("portal.portal_my_home", values)
 
+    # INICIO DEL CODIGO AGREGADO POR TRESCLOUD
+    def update_my_account_partner_fields(self, values):
```

Comment:
no existe para esto el método _prepare_portal_layout_values() ??

Reply:
No nos sirve. Es método se usa para cuando se entra a '/my/account' a ver los valores del usuario. En nuestro caso necesitamos cambiar valores cuando se hace el post para modificar dichos valores. Nótese que en la línea 185 cuando es post la variable value toma otro valor totalmente distinto:
values = {key: post[key] for key in self.MANDATORY_BILLING_FIELDS}